build/makelib/golang.mk

```diff
@@ -185,11 +179,6 @@ go.mod.clean:
 	@sudo rm -fr $(WORK_DIR)/cross_pkg
 	@$(GOHOST) clean -modcache
 
-$(GOLINT):
-	@echo === installing golint
-	@mkdir -p $(TOOLS_HOST_DIR)/tmp
-	@GOPATH=$(TOOLS_HOST_DIR)/tmp GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) get github.com/golang/lint/golint
-	@rm -fr $(TOOLS_HOST_DIR)/tmp
 
```

Comment:
This is going to leave 2 blank lines between targets.

Reply:
actually no. It leaves only one blank line. This time I double-checked before committing and pushing 😄

Reply:
This is because mt patch also removes one blank line, leaving only a single one above the removed hunk.

Reply:
Line 181 and 182 are blank after the patch. Do you have an un-pushed local change?

Reply:
gosh, sorry for being such a smart Alec! 😉  I had missed that. fixed now